operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -7,6 +7,49 @@
 # the pod should be restarted. The script checks a WebLogic Server state file which
 # is updated by the node manager.
 
+function copySitCfgWhileRunning() {
+  # Helper fn to copy sit cfg xml files to the WL server's domain home.
+  #   - params $1/$2/$3 == 'src_dir tgt_dir fil_prefix'
+  #   - $src_dir files are assumed to start with $fil_prefix and end with .xml
+  #   - copied $tgt_dir files are stripped of their $fil_prefix
+  #   - any .xml files in $tgt_dir that are not in $src_dir/$fil_prefix+FILE are deleted
+  #
+  # This method is called while the server is already running, see 
+  # 'copySitCfgWhileBooting' in 'startServer.sh' for a similar method that
+  # is called just before the server boots.
+  #
+  # It will do nothing unless the environment variable DYNAMIC_CONFIG_OVERRIDE is set.
+
+if [ ! "${DYNAMIC_CONFIG_OVERRIDE:-notset}" = notset ]; then
+    (
+    shopt -s nullglob  # force file matching 'glob' for loops below to run 0 times if 0 matches
+    local src_dir=${1?}
+    local tgt_dir=${2?}
+    local fil_prefix=${3?}
+    local local_fname
+    local tgt_file
+    trace "Copying files starting with '$src_dir/$fil_prefix' to '$tgt_dir' without the prefix."
```

Comment:
Does this trace have a side effect on eventing/logging?  This will be called every 10 seconds, which seems distracting if so.    Maybe only trace at line 38 (just before the cp) to only trace when copying a change over?

Reply:
That's in a loop, isn't it? Wouldn't that result in multiple log messages?

Reply:
The liveness probe is called every  10 seconds.  So this trace will occur every 10 seconds in theory somewhere.   I'm suggesting moving the trace down to the copy - which only occurs when the file in question changed - so yes, multiple log messages, but only when a file changes instead of every 10 seconds...

Reply:
Agreed, although without an opinion on exactly where the trace belongs, as a trace every 10 seconds will overwhelm other logging.

Reply:
Looks fixed.